src/api/TourneyRent/TourneyRent.Presentation.Api/Views/Team/TeamRead.cs

```diff
@@ -0,0 +1,15 @@
+﻿using TourneyRent.DataLayer.Models;
+
+namespace TourneyRent.Presentation.Api.Views.Team
+{
+    public class TeamRead
```

Comment:
I think it would be better to add **View** suffix for these e.g. TeamView, just to make it more clear that this class should be used in the Presentation layer.